### DISCUSSION TITLE
Auto install  MySQL database

### DISCUSSION THREAD

**Discussion by mexup** [IceWhaleTech/CasaOS#325] [Ideas]
It will be nice if you guys please add auto MySQL install in all store, will will help to add database for other apps like WordPress or NextCloud

Thank you!

**Comment by Hedius** [comment]
I would prefer MariaDB over mysql in this case.

Most distributions already moved away from MySQL and replaced the package with MariaDB and in respect of the open source community it would be good to also reflect this change here.

**Comment by Tobias-Lauerbach** [comment]
I also prefer MariaDB

**Comment by Tobias-Lauerbach** [comment]
I also had the idear of adding a MySQL/MariaDB darabase as a system feature because many Docker containers need one and it would make it much more easy, aspechealy if there would be an grafical interface to do Basic Database Edits, like Adding a User / changing passwords / edit the priveleges/rights of the user
Adding a Database
showing Databases
...

**Reply by Hedius** [reply]
Eh you can use phmyadmin for that. And install it as a docker container

**Reply by Tobias-Lauerbach** [reply]
I looked at it but  I think a MySQL based Database is so fundanental for many Docker images (like Nextcloud / NginxProxyManeger / ...) that it would be greate to have an build in Database which is easy to use and maybe even without any terminal use required, just backed in the UI, I think that would enhance the user experience alot, would make CasaOS stand out.
